### DISCUSSION TITLE
[Windows] Rust Debuggables throws error when `vim.o.sh='pwsh'`

### DISCUSSION THREAD

**Discussion by elithrade** [mrcjkb/rustaceanvim#330] [Q&A] [answered]
I have LazyVim setup on my Windows machine, I enabled extras for rust and DAP. I pressed <leader>dr to bring up the debuggable popup and selected `build --package <package> --bin <package> --all-feature` and expecting debugger will start and my breakpoint will hit. Instead I got the following error:
![image](https://github.com/mrcjkb/rustaceanvim/assets/10831354/e5b42292-f4cd-442e-9d88-a560334b91d8)

I have exactly the same config in WSL and it can pick up the binary and my breakpoint hit.

Am I doing something wrong on Windows machine? I can run `cargo build` and `cargo run` no problem on Windows, why is the debuggable complaining it cannot compile?

Here is my health check, eveything is OK.
![image](https://github.com/mrcjkb/rustaceanvim/assets/10831354/4b7e27d5-4b65-47b0-8d98-c032af9c71c6)

Thanks.

**Comment by mrcjkb** [comment]
Hey :wave:

I've just pushed a change that includes `stderr` and `stdout` in the error message, if present, as well as the command that was run to compile the debug build.

Could you please update and try again?
It'll still fail, but we'll hopefully have some more information about why.

**Comment by elithrade** [comment]
Thanks you for the quick reply, I got the latest and got this error:
![image](https://github.com/mrcjkb/rustaceanvim/assets/10831354/0c92aa4d-1fde-45cb-8f7b-fbe50c6426a2)
If I run the same `cargo build --package test_debugger --bin test_debugger --all-features --message-format=json` command in terminal and it works.
I am still earning rust, anything I am missing here?
Thanks.

**Reply by mrcjkb** [reply, marked as answer]
🤔 looks like it's related to PowerShell.

https://github.com/neovim/neovim/issues/13893

**Reply by elithrade** [reply]
Thanks you so much, I updated my nvim config as per https://github.com/neovim/neovim/issues/13893#issuecomment-1409668526 and it is now working.
